_posts/2021-01-07-debezium-1-4-final-released.adoc

```diff
@@ -0,0 +1,75 @@
+
+* New link:/documentation/reference/connectors/vitess.html[Vitess] connector
+* Fine-grained Selection of Snapshotted Tables
+* PostgreSQL `Snapshotter` completion hook
+* Distributed link:/blog/2020/12/16/distributed-tracing-with-debezium/[Tracing]
+* MySQL support for _create_ or _read_ records emitted during snapshot
+* Many Oracle link:/documentation/reference/connectors/oracle.html#_logminer[Logminer adapter] improvements
+* Full support for Oracle JDBC connection strings
+* Improved reporting of DDL errors
+
++++<!-- more -->+++
+
+Please refer to previous release announcements (https://debezium.io/blog/2020/10/23/debezium-1-4-alpha1-released/[Alpha1], https://debezium.io/blog/2020/11/17/debezium-1-4-alpha2-released/[Alpha2], https://debezium.io/blog/2020/12/09/debezium-1-4-beta1-released/[Beta1], https://debezium.io/blog/2020/12/17/debezium-1-4-cr1-released/[CR1]) for more details.
+Since the CR1 release just before the holidays, we've https://issues.redhat.com/issues/?jql=project%20%3D%20DBZ%20AND%20fixVersion%20%3D%201.4.0.Final%20ORDER%20BY%20issuetype%20DESC%2C%20updated%20DESC%2C%20priority%20DESC[focused] on addressing some remaining bugs and improvements.
+
+Thank you to everyone involved in testing the previous releases, this is invaluable by spotting and addressing any problems with new features as well as regressions.
+And of course we'd like to thank all the community members contributing to this release:
+https://github.com/alisator[Alisa Houskova],
+https://github.com/ani-sha[Anisha Mohanty],
+https://github.com/AndreyIg[Andrey Ignatenko],
+https://github.com/bingqinzhou[Bingqin Zhou],
+https://github.com/bduisenov[Babur Duisenov],
+https://github.com/creactiviti[Arik Cohen],
+https://github.com/Faizan[Faizan],
+https://github.com/grantcooksey[Grant Cooksey],
+https://github.com/hauntingEcho[Matt Beary],
+https://github.com/hussain-k1[Mohamed Pudukulathan],
+https://github.com/morozov[Sergei Morozov],
+https://github.com/keweishang[Kewei Shang],
+https://github.com/mans2singh[Mans Singh],
+https://github.com/martper2[Martin Perez],
+https://github.com/michaelwang[Michael Wang],
+https://github.com/Iskuskov[Alexander Iskuskov]
+https://github.com/jgormley6[James Gormley],
+https://github.com/jinguangyang[jinguangyang],
+https://github.com/KaushikIyer16[Kaushik Iyer],
+https://github.com/jeremy-l-ford[Jeremy Ford],
+https://github.com/johnjmartin[John Martin],
+https://github.com/ramanenka[Vadzim Ramanenka],
+https://github.com/rareddy[Ramesh Reddy],
+https://github.com/rgannu[Ganesh Ramasubramanian],
+https://github.com/seeekr[Denis Andrejew],
+https://github.com/telnicky[Travis Elnicky],
+https://github.com/vanhoale[Hoa Le],
+https://github.com/yimingl17[Yiming Liu],
+https://github.com/yrodiere[Yoann Rodière], and
+https://github.com/zrlurb[Peter Urbanetz].
+
+Overall, more than 220 individuals have contributed to the Debezium project and the number of Debezium link:/community/users/[users] continues to grow.
```

Comment:
I got 245 names in the contributors file, so let's use that number? You also could link to yesterday's post btw.